yarn-project/ethereum/src/deploy_l1_contracts.ts

```diff
@@ -234,13 +236,34 @@ export const deployL1Contracts = async (
   ]);
   logger.info(`Deployed Rollup at ${rollupAddress}`);
 
+  const rollup = getContract({
+    address: getAddress(rollupAddress.toString()),
+    abi: contractsToDeploy.rollup.contractAbi,
+    client: walletClient,
+  });
+
+  // @note  We make a time jump PAST the very first slot to not have to deal with the edge case of the first slot
```

Comment:
can you explain why the first slot is such an edge case, not necessarily in the code, but to me

Reply:
The genesis block is at slot 0, so the slot is already occupied.

Reply:
update: necessarily in the code